src/ui/components/BatteryStatus.tsx

```diff
@@ -18,9 +18,12 @@ export const BatteryStatus: FC<BatteryStatusProps> = ({
   const Text = variant === 'large' ? LargeText : SmallText;
 
   return batteryLevel ? (
-    <Row alignItems="center" flex={1}>
-      <Icon.Battery isCharging={isCharging} batteryLevel={batteryLevel} size={iconSize} />
+    <Row>
       <Text>{String(batteryLevel)}</Text>
+      <Row style={{ marginLeft: 10 }} />
```

Comment:
Worth adding this to `Style.ts` constants?

Reply:
I've actually gone off the style constants quite a bit. How do you feel about it?